Droid/MainApplication.cs

```diff
@@ -20,19 +19,17 @@ public class MainApplication : Application
         // However, if you do not specify the license key when initializing the SDK, 
         // it will work in trial mode (trial period of 1 minute). 
         // To get another trial period you have to restart your app.
-        const string licenseKey = "";
-
+        const string licenseKey = null;
 
         public MainApplication(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
         { }
 
-
         public override void OnCreate()
         {
             base.OnCreate();
 
             Log.Debug(LOG_TAG, "Initializing Scanbot SDK...");
-            SBSDK.Initialize(this, licenseKey, true);
+            SBSDKInitializer.Initialize(this, licenseKey, new SBSDKConfiguration { EnableLogging = true });
```

Comment:
The initialization of the SDK should also be done on the Xamarin.Forms layer (in the forms project). Ideally, the developer would not touch the native layer (Droid and iOS projects).